babel-node -b <preset> cli flag doesn't work

## Bug Report

when using `babel-node --presets <preset> <file>` everything works but when using the shorthand variant `babel-node -b <preset> <file>` it doesn't.

```sh
/tmp/test $ ./node_modules/.bin/babel-node --presets @babel/preset-env test.js 
[ 1, 2, 3, 4 ]
/tmp/test $ ./node_modules/.bin/babel-node -b @babel/preset-env test.js 
internal/modules/cjs/loader.js:605
    throw err;
    ^

Error: Cannot find module '/private/tmp/test/@babel/preset-env'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:603:15)
    at Function.Module._load (internal/modules/cjs/loader.js:529:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:775:12)
    at Object.<anonymous> (/private/tmp/test/node_modules/@babel/node/lib/_babel-node.js:234:23)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:733:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:775:12)

```

**Expected behavior/code**

`babel-node -b <preset>` should work excactly the same as `babel-node --presets <preset>`, according to `babel-node --help`.

**Babel Configuration (.babelrc, package.json, cli command)**

package.json:

```json
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@babel/cli": "^7.2.3",
    "@babel/core": "^7.2.2",
    "@babel/node": "^7.2.2",
    "@babel/preset-env": "^7.2.3"
  },
  "dependencies": {
    "uniq": "^1.0.1"
  }
}
```

input `test.js`:
```js
import uniq from 'uniq'
console.log(uniq([1, 2, 3, 4]))

```


**Environment**
macos 10.13
npm 6.5.0

--- a/packages/babel-node/package.json
+++ b/packages/babel-node/package.json
@@ -20,7 +20,7 @@
   ],
   "dependencies": {
     "@babel/register": "^7.7.7",
-    "commander": "^2.8.1",
+    "commander": "^4.0.1",
     "core-js": "^3.2.1",
     "lodash": "^4.17.13",
     "node-environment-flags": "^1.0.5",

--- a/packages/babel-node/src/_babel-node.js
+++ b/packages/babel-node/src/_babel-node.js
@@ -169,15 +169,15 @@ if (program.eval || program.print) {
       }
 
       if (arg[0] === "-") {
-        const camelArg = arg
-          .slice(2)
-          .replace(/-(\w)/, (s, c) => c.toUpperCase());
-        const parsedArg = program[camelArg];
-        if (
-          arg === "-r" ||
-          arg === "--require" ||
-          (parsedArg && parsedArg !== true)
-        ) {
+        const parsedOption = program.options.find(option => {
+          return option.long === arg || option.short === arg;
+        });
+        if (parsedOption === undefined) {
+          return;
+        }
+        const optionName = parsedOption.name();
+        const parsedArg = program[optionName];
+        if (optionName === "require" || (parsedArg && parsedArg !== true)) {
           ignoreNext = true;
         }
       } else {

--- a/packages/babel-node/src/_babel-node.js
+++ b/packages/babel-node/src/_babel-node.js
@@ -175,7 +175,7 @@ if (program.eval || program.print) {
         if (parsedOption === undefined) {
           return;
         }
-        const optionName = parsedOption.name();
+        const optionName = parsedOption.attributeName();
         const parsedArg = program[optionName];
         if (optionName === "require" || (parsedArg && parsedArg !== true)) {
           ignoreNext = true;

--- a/packages/babel-node/test/fixtures/babel-node/--presets/in-files/index.js
+++ b/packages/babel-node/test/fixtures/babel-node/--presets/in-files/index.js
@@ -1,0 +1,1 @@
+console.log("foo");

--- a/packages/babel-node/test/fixtures/babel-node/--presets/in-files/presetFile.js
+++ b/packages/babel-node/test/fixtures/babel-node/--presets/in-files/presetFile.js
@@ -1,0 +1,4 @@
+module.exports = function () {
+  console.log("Preset was loaded, so --presets was used.");
+  return {};
+};

--- a/packages/babel-node/test/fixtures/babel-node/--presets/options.json
+++ b/packages/babel-node/test/fixtures/babel-node/--presets/options.json
@@ -1,0 +1,3 @@
+{
+  "args": ["--presets", "./presetFile.js", "index"]
+}

--- a/packages/babel-node/test/fixtures/babel-node/--presets/stdout.txt
+++ b/packages/babel-node/test/fixtures/babel-node/--presets/stdout.txt
@@ -1,0 +1,2 @@
+Preset was loaded, so --presets was used.
+foo

--- a/packages/babel-node/test/fixtures/babel-node/-b/in-files/index.js
+++ b/packages/babel-node/test/fixtures/babel-node/-b/in-files/index.js
@@ -1,0 +1,1 @@
+console.log("foo");

--- a/packages/babel-node/test/fixtures/babel-node/-b/in-files/presetFile.js
+++ b/packages/babel-node/test/fixtures/babel-node/-b/in-files/presetFile.js
@@ -1,0 +1,4 @@
+module.exports = function () {
+  console.log("Preset was loaded, so -b was used.");
+  return {};
+};

--- a/packages/babel-node/test/fixtures/babel-node/-b/options.json
+++ b/packages/babel-node/test/fixtures/babel-node/-b/options.json
@@ -1,0 +1,3 @@
+{
+  "args": ["-b", "./presetFile.js", "index"]
+}

--- a/packages/babel-node/test/fixtures/babel-node/-b/stdout.txt
+++ b/packages/babel-node/test/fixtures/babel-node/-b/stdout.txt
@@ -1,0 +1,2 @@
+Preset was loaded, so -b was used.
+foo

